### ISSUE TITLE
0.8.0 breaks launch_block_device_mappings for gp2 volumes

### ISSUE THREAD

**Issue by ralfthewise** [mitchellh/packer#2369]
Here's my `packer.json` builder config:

```
      "builders": [
        {
          "name": "ec2-radar-postgres-server",
          "ami_name": "radar-postgres-{{user `replication_role`}}-{{timestamp}}",
          "ami_description": "Postgres server",
          "type": "amazon-ebs",
          "access_key": "{{user `aws_access_key`}}",
          "secret_key": "{{user `aws_secret_key`}}",
          "region": "{{user `aws_region`}}",
          "source_ami": "{{user `source_ami`}}",
          "instance_type": "m3.medium",
          "ssh_username": "{{user `ssh_username`}}",
          "ssh_timeout": "10m",
          "security_group_ids": ["{{user `security_group_id`}}"],
          "vpc_id": "{{user `vpc_id`}}",
          "subnet_id": "{{user `subnet_id`}}",
          "iam_instance_profile": "PostgresServer",
          "associate_public_ip_address": true,
          "launch_block_device_mappings": [
            {
              "device_name": "/dev/sdf",
              "volume_type": "gp2",
              "volume_size": 128,
              "delete_on_termination": true,
              "encrypted": true
            }
          ]
        }
      ]
```

here is the output of building it:

```
Debug mode enabled. Builds will not be parallelized.
ec2-radar-postgres-server output will be in this color.

==> ec2-radar-postgres-server: Prevalidating AMI Name...

[logs removed for brevity]

Build 'ec2-radar-postgres-server' errored: Error launching source instance: InvalidParameterCombination: The parameter iops is not supported for gp2 volumes.
	status code: 400, request id: []

==> Some builds didn't complete successfully and had errors:
--> ec2-radar-postgres-server: Error launching source instance: InvalidParameterCombination: The parameter iops is not supported for gp2 volumes.
	status code: 400, request id: []

==> Builds finished but no artifacts were created.
```

**Comment by catsby** [comment]
Hey @ralfthewise – this was patched in https://github.com/mitchellh/packer/pull/2197 and released in Packer v0.8.1, can you try updating your Packer install?

I'm going to close for now, please let me know if this isn't resolved for you in the latest release

**Comment by ralfthewise** [comment]
Thanks, that fixed it!
